app/helpers/buttons_helper.rb

```diff
@@ -142,6 +144,17 @@ def dropdown_link(*args)
     end
   end
 
+  PATTERNFLY_LINK_CLASS = 'pf-c-button pf-m-link'
+
+  %w[link_to action_link_to fancy_button_to fancy_link_to delete_button_for delete_link_for].each do |method_sym|
```

Comment:
Sorry. My fault actually, but better:

```suggestion
  %i[link_to action_link_to fancy_button_to fancy_link_to delete_button_for delete_link_for].each do |method_sym|
```